src/ert/gui/model/snapshot.py

```diff
@@ -295,11 +295,12 @@ def _add_snapshot(self, snapshot: Snapshot, iter_: int):
                 NodeType.REAL,
             )
             snapshot_tree.add_child(real_node)
-            for step_id, step in real[ids.STEPS].items():
-                step_node = Node(step_id, {ids.STATUS: step[ids.STATUS]}, NodeType.STEP)
+            for step_id, step in snapshot.steps(real_id).items():
+                step_node = Node(step_id, {ids.STATUS: step.status}, NodeType.STEP)
                 real_node.add_child(step_node)
                 for job_id in metadata[SORTED_JOB_IDS][real_id][step_id]:
-                    job = step[ids.JOBS][job_id]
+                    job = snapshot.get_job(real_id, step_id, job_id)
+                    # job = step[ids.JOBS][job_id]
```

Comment:
Supposing this line should be taken out.

Reply:
yassss : D thanks for catching this!

it's already gone though.